### DISCUSSION TITLE
Getting started with USWDS proposal discussions

### DISCUSSION THREAD

**Discussion by thisisdano** [uswds/uswds#5764] [Proposals]
Welcome to the proposal discussion board! Use this space to suggest and discuss ideas for [new components and patterns](https://designsystem.digital.gov/components/lifecycle/) for USWDS.

## Who can contribute

Anyone can suggest a component or pattern and contribute to proposal discussions. To participate in these discussions, you must agree to follow our [community guidelines](https://designsystem.digital.gov/about/community/#community-conduct).

Participating in GitHub Discussions requires a [GitHub account](https://github.com/login). If you're not able to get a GitHub account, [reach out to us via email](mailto:uswds@gsa.gov), and we will add your contribution to the discussion board.

### Opening a new proposal discussion

If you have a new component or pattern idea, get a conversation started with a new discussion. The USWDS team uses these proposal discussions to create formal proposals. 

1. First, review the [proposals category](https://github.com/uswds/uswds/discussions/categories/proposals) in GitHub discussions to see if anyone has already suggested your idea. If your solution has been suggested, you can share ideas and add information or context to the discussion, or just express your interest.
1. If a discussion doesn’t exist, create a new discussion. This discussion is the place to discuss the merits of the new solution, in collaboration with the community and the USWDS team. What makes a good case for a USWDS component or pattern? We've outlined the criteria in our USWDS proposal template.
1. Use [USWDS GitHub Discussions](https://github.com/uswds/uswds/discussions) to get feedback and support from peers. 
1. If the discussion tends toward support for including your solution in the design system, the USWDS core team collects this information to create a formal proposal. The formal proposal will include all information necessary to publish it on the USWDS site.
1. The USWDS team will share the formalized proposal in that discussion thread. When it's complete, there'll be a 45-day comment period for the community to provide feedback on the proposal.
1. Finally, the USWDS core team will evaluate the proposal and update you and the community about the outcome with further comments in that discussion.

### Contributing to an existing discussion
Every discussion can benefit from community input. If you’d like to contribute to existing ideas, we encourage you to: 
- Vote for ideas
- Add information, research, and solutions
- Share opinions and ask questions

### Vote for proposal discussions
One of the easiest and most effective ways to express your interest in an idea is to vote. Proposal discussions with more votes are likely to be reviewed more quickly by the USWDS core team. 

#### How to vote
Use the [upvote](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/participating-in-a-discussion#upvoting-a-discussion) feature in the discussion board to vote for ideas that you want to see (and would use!) in the design system. You can also upvote discussion comments that you agree with.

Once the USWDS core team has shared a formal proposal on a discussion thread, a vote on the proposal will indicate that you agree with the proposal and believe it’s ready for formal evaluation. Remember to vote here in GitHub discussions, not in the [proposal repo](https://github.com/uswds/uswds-proposals).

### Add information, research, and solutions
You can help advance existing discussions by adding research, context, solutions, etc., in the comments. Any relevant information that you add to a discussion can help make the case for evaluating an idea and including it in USWDS. See the discussion topics below to understand the types of information we are looking for. 

### Share opinions and ask questions
We encourage you to ask questions and share your thoughts in the proposal discussion; the community’s thoughts and opinions play a critical role in this process. Please note that all discussions must follow our [community guidelines](https://designsystem.digital.gov/about/community/#community-conduct). You can participate in any discussion before or during the 45-day comment period.

## What your discussion needs
Any suggestion should address the points below before the USWDS core team will evaluate it. It helps if you link to supporting evidence and research wherever you can. We ask for a lot of detail, but it’s okay to start with basic information — whatever you have available. You can work with the community to add more detail over time.

- **Why USWDS needs this:** Include key interactions, current gaps in USWDS, and the problems this solution solves. Explain how it supports federal laws, guidance, or policies.
- **How the solution works:** Include any interactive states and variants. Diagrams, images, wireframes, and prototypes can help you make your case.
- **Real-life examples:** Link to examples of other sites — preferably government sites — that use this kind of solution.
- **When to use this solution and when to consider something else:** For example, what kind of content works best with this component or pattern, and what kind of content would work poorly with it? What are common misuses of this type of solution?
- **Usability considerations:** Include conditions that would make this solution more usable or less usable. Also include common pitfalls or implementation mistakes associated with it.
- **Accessibility considerations:** Include how this component interacts with assistive technologies such as keyboard navigation, screen readers, zoom magnification, and voice command. Are there audiences that might struggle with this solution? How might it become inaccessible?
- **Who’s involved:** Please tag volunteers who may be willing to help design, develop or test this component. Note agencies that can offer significant support for the solution, and tag any stakeholders. Be sure to share a list of those who helped research and prepare this proposal. Include your name and/or GitHub username if you’d like credit as a contributor.

Once a proposal discussion has addressed all of these, the USWDS core team will create a formal proposal, open that proposal for community feedback, then evaluate it. We’ll share our decision with the community on the relevant discussion page. You can check the status of any proposal on the [USWDS lifecycle status page](https://designsystem.digital.gov/components/status/).

## Helpful resources
- [Contributing to USWDS](https://designsystem.digital.gov/about/contribute/)
- [Code of conduct for this discussion board](https://designsystem.digital.gov/about/community/#community-conduct)

**Comment by ryparker** [comment]
I'm bringing the unresolved question from [#6178](https://github.com/uswds/uswds/issues/6178) into this discussion, where contributors already look for proposal guidance.

The introduction describes collaboration with the community and the USWDS team, while the checklist says suggestions should address the required points before the team evaluates them. Contributors still need to know when they can expect the team to join the conversation and what the community is expected to prepare first.

We still need to agree on when the team offers feedback, what is needed before formal evaluation, and who develops the information for a formal proposal. Once those expectations are agreed, this pinned introduction should be updated to explain them. The existing text does not settle that engagement question.
